guide/yaml/custom-entities.md

```diff
@@ -82,6 +82,18 @@ to be specified up one level, alongside the type.
 However config keys which are *not* declared on the type *must* be declared in the `brooklyn.config` block. 
 
 
+### Passing custom variables
+
+Blueprint scripts can be parametrized through environment variables, making them reusable in different use-cases.
+Define the variables in the `env` block and then reference them using the standard bash notation:
```

Comment:
Used env to be consistent with the other examples on the page. "env" is used when not using "brooklyn.config". "shell.env" is used when nested in "brooklyn.config" (that's the original idea, currently they are more or less interchangeable).

We should be consistently using "brooklyn.config" but that's for another PR.